tests/tag.rs

```diff
@@ -13,12 +13,6 @@ fn serialize_raw() {
     assert_eq!(serialized.to_vec(), 2137_i64.to_be_bytes().to_vec())
 }
 
-#[test]
-fn serialize_named() {
-    let serialized = NbtTag::Long(2137).serialize_named("hi");
-    assert_eq!(serialized.to_vec(), b"\x04\0\x02hi\0\0\0\0\0\0\x08\x59")
-}
-
```

Comment:
why did you remove this?

Reply:
because `serialize_named` was removed so there is nothing to test